src/components/calculator/PersonRiskControls.tsx

```diff
@@ -70,6 +70,53 @@ const personRiskPopover = (
   </Popover>
 )
 
+interface DurationInputProps {
+  value: number
+  onChange: (duration: number) => void
+}
+
+const DurationInput: React.FunctionComponent<DurationInputProps> = (
+  props: DurationInputProps,
+): JSX.Element => {
+  const [hours, setHours] = useState(Math.floor(props.value / 60))
+  const [minutes, setMinutes] = useState(props.value % 60)
+  const calculateDuration = (hours: number, minutes: number) => {
+    const h = Number.isNaN(hours) ? 0 : hours
+    const m = Number.isNaN(minutes) ? 0 : minutes
+    return h * 60 + m
+  }
+  return (
+    <InputGroup className="mb-3">
+      <input
```

Comment:
https://react-bootstrap.github.io/components/input-group/#input-group Apparently a Form.Control does the right thing with the CSS so that the InputGroup.Text has the corners rounded in the right places so that the input and the text look properly paired together. I wonder if there's a way to keep the stuff that makes this input have a keypad input on mobile *and* fix up the styling? Can `<Form.Control>` pass on the parameters it doesn't understand to the underlying `<input>`?

Reply:
IIUC a `Form.Control` is just an `input` with a `form-control` class, so this is already doing the thing